openedx/core/djangoapps/service_status/tasks.py

```diff
@@ -4,12 +4,11 @@
 
 import time
 
+from celery import task
 from dogapi import dog_stats_api
 
-from djcelery import celery
 
-
-@celery.task
+@task
```

Comment:
Probably. Since we're going to be unable to remove the djcelery dependence entirely, I was mostly checking to see if the "simple" removal would work out. I've reverted the changes to openedx/core/djangoapps/service_status/, so the results backend will be the only real change.